### ISSUE TITLE
Add art to WordPress.org repo

### ISSUE THREAD

**Issue by danielbachhuber** [lcache/wp-lcache#55]
We look a bit bleak:

![image](https://cloud.githubusercontent.com/assets/36432/18475293/ba3d72b4-797a-11e6-92f3-8313078d2e2e.png)

WP Redis is quite nice:

![image](https://cloud.githubusercontent.com/assets/36432/18475274/ab80a3b8-797a-11e6-9ce1-d916d241b813.png)

**Comment by danielbachhuber** [comment]
@joshkoenig Can you line this up?

**Comment by joshkoenig** [comment]
Yes, though we will want to take a minute to brand this properly across all
channels. Bleak is ok for where we are right now. ;)

**Comment by ramontolkamp** [comment]
@danielbachhuber Not sure what you have in mind for the artwork ...
As a thank you for your help I created a banner for your LCAche plugin. Hope you'll enjoy, but do with it what you like.

![wp_lcache-banner-772x250](https://user-images.githubusercontent.com/32372369/31041436-983525de-a595-11e7-885a-a9983415da7c.jpg)
![wp_lcache-banner-1544x500](https://user-images.githubusercontent.com/32372369/31041437-9837a48a-a595-11e7-89eb-0cd4c4a069fd.jpg)
